### ISSUE TITLE
After including node-sass the ng server failed with malloc error

### ISSUE THREAD

**Issue by sivaah** [angular/angular-cli#1291]
1. OS?  
Mac OSX (10.8.5)

2. Versions. Please run `ng --version`. If there's nothing outputted, please
```
angular-cli: 1.0.0-beta.8
node: 4.4.3
os: darwin x64
```
3. Repro steps. Was this an app that wasn't created using the CLI? What change did you do on your code? etc.

`npm install node-sass`

4. The log given by the failure. Normally this include a stack trace and some more information.

```
$ ng serve
Livereload server on http://localhost:49152
Serving on http://localhost:4200/
angular-cli(34873,0x7fff7c63d180) malloc: *** error for object 0x10340b06c: pointer being freed was not allocated
*** set a breakpoint in malloc_error_break to debug
Abort trap: 6
```

5. Mention any other details that might be useful.

Removing the node-sass module fixes the problem
`npm uninstall node-sass
`
Unable to use node-sass in angular-cli.

**Comment by jkuri** [comment]
Interesting. @hansl can you take a look at this one maybe?

**Comment by filipesilva** [comment]
Closed as obsolete following #1455.
